server/tests/api/test_points.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from src.db.db_utils import exec_sql_file
+from tests.test_utils import *
+
+
+class TestPoints(unittest.TestCase):
+    def setUp(self):
+        """
+        Initialize mock API calls and DB setup
+        """
+        self.mock_post_rest_call = patch('tests.test_utils.post_rest_call').start()
+        self.mock_get_rest_call = patch('tests.test_utils.get_rest_call').start()
+        self.mock_delete_rest_call = patch('tests.test_utils.delete_rest_call').start()
+        self.mock_put_rest_call = patch('tests.test_utils.put_rest_call').start()
+
+    def tearDown(self):
+        """
+        Stop mocking
+        """
+        patch.stopall()
+
+    def test_update_points(self):
+        """
+        Test updating points for a user
+        """
+        new_vals = {"points": "999"}
+        self.mock_post_rest_call.return_value = {"message": "Success"}
+        result = self.mock_post_rest_call("http://localhost:8080/points/069b7b35-fbcf-427b-9957-aad6a718e410",params=new_vals,)
```

Comment:
Based on the change suggested in the api file, this would be `mock_put_rest_call`